INSTALLATION_GUIDE.md

```diff
@@ -0,0 +1,41 @@
+## Docker enviroment
+
+### Installation
+
+Docker allows you to run identical software on all platforms. In
+Windows, in particular, there are challenges ensuring all python
+packages are exactly the same as those available on other platforms.
```

Comment:
I think we're missing a "so that it why..." kind of context here. Imagining I'm a newbie I might read this as "Docker allows you to.... but Windows makes it hard"  rather than "windows makes it hard, so Docker makes it easy to..."

Reply:
see new version